src/components/SignUpForm.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { Grid, Button, Form, Input } from "semantic-ui-react";
+import auth from "../modules/auth";
+import { useHistory } from "react-router-dom";
+import "../css/index.css";
+
+const SignUpForm = (props) => {
+  const [message, setMessage] = useState("");
+  const history = useHistory();
+
+  const signup = async (e) => {
+    e.preventDefault();
+    try {
+      let response = await auth.signUp({
+        email: e.target.email.value,
+        password: e.target.password.value,
+        password_confirmation: e.target.passwordConfirmation.value,
+      });
+      if (response.data.success) {
+        props.setUid(response.data.uid);
+        history.push("/sign_in");
+      }
+    } catch (error) {
+      setMessage(error.response.data.errors[0]);
+    }
+  };
+  return (
+    <>
+      <Grid className="signup-container" verticalAlign="middle">
+        <Grid.Column align="center">
+          <h3 id="error-message">{message}</h3>
+          <Form unstackable id="signup-form" onSubmit={signup}>
+            <h1>Sign up</h1>
+            <h4>Email</h4>
+            <Input
+              name="email"
+              type="email"
+              id="email"
+              placeholder="email"
+            ></Input>
+            <h4>Password</h4>
+            <Input
+              name="password"
+              type="password"
+              id="password"
+              placeholder="Password"
```

Comment:
We are not using placeholders in the login form. We should be consistent with styling across similar forms. As we have a label for the field, the placeholder isn't really necessary.

Reply:
ok then i remove them, but i do like to have them...